tensorboard/util/test_util.py

```diff
@@ -25,11 +25,17 @@
 from absl import logging
 import contextlib
 import functools
+import inspect
 import os
+import six
 import sqlite3
 import threading
+import unittest
+from unittest import loader as _unittest_loader
+
 
 import tensorflow as tf
+from tensorflow.python import tf2
```

Comment:
It seems like we need then fix the problem elsewhere too.
Since I am more interested in enabling v2 tests right now, I will let future commits fix things like `tensorboard.compat.tf2` (or is that version correct as is? Even considering the env variable?)